src/ProxyClient/Invalidation/TagsInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\ProxyClient\Invalidation;
+
+use FOS\HttpCache\ProxyClient\ProxyClientInterface;
+
+/**
+ * An HTTP cache that supports invalidation by a cache tag, that is, removing, or expiring
+ * objects from the cache tagged with a given tag or set of tags.
+ */
+interface TagsInterface extends ProxyClientInterface
+{
+    /**
+     * Remove/Expire cache objects based on cache tags
+     *
+     * @param array $tags Tags that should be removed/expired from the cache
+     * @param string $tagsHeader Name of the HTTP header used to provide the tags.
+     *
+     * @return $this
+     */
+    public function tags(array $tags, $tagsHeader = 'X-Cache-Tags');
```

Comment:
i think we should call this invalidateTags or something. the other methods are nice short ban/purge/refresh, but all of those words are verbs, while tag is not a verb and does not say whether you want to add tags to something or to invalidate the tags. i prefer verbose over implicit/contextual here.